tests/partition_selection_test.py

```diff
@@ -67,6 +67,27 @@ def test_truncated_pre_thresholding(self, mock_method):
         mock_method.assert_called_once_with("gaussian", eps, delta,
                                             max_partitions, pre_threshold)
 
+    @patch("pydp.algorithms.partition_selection.create_partition_strategy")
+    def test_truncated_laplace_thresholding_with_sigma(self, mock_method):
+        sigma, delta, max_partitions_contributed, pre_threshold = 1.5, 1e-5, 2, 3
+        partition_selection.create_laplace_thresholding(
+            sigma, delta, max_partitions_contributed, pre_threshold)
+        mock_method.assert_called_once_with("laplace",
+                                            math.sqrt(2) / sigma, delta,
+                                            max_partitions_contributed,
+                                            pre_threshold)
+
+    @patch("pydp.algorithms.partition_selection.create_partition_strategy")
+    def test_truncated_gaussian_thresholding_with_sigma(self, mock_method):
+        sigma, delta, max_partitions_contributed, pre_threshold = 1.5, 1e-5, 5, 10
+        partition_selection.create_gaussian_thresholding(
+            sigma, delta, max_partitions_contributed, pre_threshold)
+        expected_epsilon = 2.7533813795016613
```

Comment:
is it useful to hardcode such a precise number? but I guess we have no choice if we assert it with mockito...maybe not assert on that? or add a comment how we calculated that.

also, it would be prefereable to test properties of the object returned by `create_gaussian_thresholding` but I guess it is not possible, or?